### PR TITLE
Update pgAdmin RBAC to match feature requirements

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [x] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
Adjust kubebuilder markers and regenerate RBAC for current pgAdmin feature.

**Other Information**:
PGO-565

### CODE REVIEW

File: internal/controller/standalone_pgadmin/controller.go [hunk 1]
```diff
@@ -81,9 +81,8 @@ func (r *PGAdminReconciler) watchPostgresClusters() handler.Funcs {
 	}
 }
 
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins,verbs={get,list,watch,create,update,patch,delete}
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/status,verbs={get,update,patch}
-//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/finalizers,verbs={update}
+//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins,verbs={get}
+//+kubebuilder:rbac:groups=postgres-operator.crunchydata.com,resources=pgadmins/status,verbs={patch}
```
Comment (cbandy) [inline internal/controller/standalone_pgadmin/controller.go:85]: ⛏️ I like the quotes in the other one.
```suggestion
//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgadmins",verbs={get}
//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgadmins/status",verbs={patch}
```

**Review by jmckulk — APPROVED** [review]
lgtm

File: internal/controller/standalone_pgadmin/postgrescluster.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgpgadmins",verbs={list}
+//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgadmins",verbs={list}
```
Comment (benjaminjb) [inline internal/controller/standalone_pgadmin/postgrescluster.go:27]: hahaha, oh, that's good.

File: internal/controller/standalone_pgadmin/controller.go (hunk 1 above)
Comment (benjaminjb) [reply internal/controller/standalone_pgadmin/controller.go:85]: Quotes is the pattern elsewhere, so 👍 to that

Comment (tjmoore4) [reply internal/controller/standalone_pgadmin/controller.go]: :+1: Updated.

**Pull request merged**